buildpack.md

```diff
@@ -719,26 +686,9 @@ Given the start command and execution strategy,
       2. Secondly, in alphabetically ascending order by layer directory name.
       3. Thirdly, in alphabetically ascending order by file name.
 
-1. If using an execution strategy involving a shell, the lifecycle MUST use a single shell process, with working directory `<app>`, to
-   1. source each file in each `<layers>/<layer>/profile.d` directory,
-      1. Firstly, in order of `/bin/build` execution used to construct the OCI image.
-      2. Secondly, in alphabetically ascending order by layer directory name.
-      3. Thirdly, in alphabetically ascending order by file name.
-   2. source each file in each `<layers>/<layer>/profile.d/<process>` directory,
-      1. Firstly, in order of `/bin/build` execution used to construct the OCI image.
-      2. Secondly, in alphabetically ascending order by layer directory name.
-      3. Thirdly, in alphabetically ascending order by file name.
-   3. source [†](README.md#linux-only)`<app>/.profile` or [‡](README.md#windows-only)`<app>/.profile.bat` if it is present.
-
-1. The lifecycle MUST set the working directory for the start command to `<working-dir>`, or to `<app>` if `<working-dir>` is not specified.
-
-1. The lifecycle MUST invoke the start command with the decided execution strategy.
-
-[†](README.md#linux-only)When executing a process using any execution strategy, the lifecycle SHOULD replace the lifecycle process in memory without forking it.
-
-[†](README.md#linux-only)When executing a process with Bash, the lifecycle SHOULD additionally replace the Bash process in memory without forking it.
+1. The lifecycle MUST use the `execve` syscall to invoke the command with its arguments, environment, and working directory following the process outlined in the [Platform Interface Specification](platform.md).
```

Comment:
This reference to the `execve` syscall is linux specific right?

Reply:
Yes, I just kept it because it was there before. IMHO, we could remove the specific syscall and have something like:

> The lifecycle MUST invoke the command with its arguments, environment, and working directory...

Without losing anything. Does that seem reasonable?

Reply:
Resolving - [moving execution details to platform spec](https://github.com/buildpacks/spec/pull/306) per the below, without the `execve` reference.